devicon.json

```
@@ -8199,6 +8199,30 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "wasm",
+        "altnames": ["WebAssembly"],
+        "tags": [
+            "binary",
+            "programming",
+            "virtual machine",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#000000",
+        "aliases": []
```

Because original and plain are the same (except for the fill color) we can remove the plain icon, and instead implement it as an alias. The `color` attribute should also be the same as the main color of the icon (unless the branding guidelines state something else). `rgb(39.215686%,30.588235%,93.72549%)` converted to hex is `#004eef`
```suggestion
            "svg": [
                "original",
                "original-wordmark",
                "plain-wordmark"
            ],
            "font": [
                "original",
                "plain-wordmark"
            ]
        },
        "color": "#004eef",
        "aliases": [
            {
                "base": "original"
                "alias": "plain"
            }
        ]
```